test/mocker.js

```diff
@@ -51,7 +51,11 @@ const normalizeUrl = function(url) {
   return url.replace(/\/{2,}/g, '/')
 }
 
-const parseReqBody = function(raw) {
+const parseReqBody = function(raw, ignoreParseBody) {
```

Comment:
why do we need ignoreParseBody? 🤔 we can simply not call parseReqBody if not required right?